src/query/models/types.go

```diff
@@ -132,9 +131,6 @@ type Tag struct {
 	Value []byte
 }
 
-// TagsTransform transforms a set of tags.
-type TagsTransform func(context.Context, []Tag) ([]Tag, error)
```

Comment:
needed to move to avoid a circular dep when adding `ClusterNamespace` to the signature